Setup/UpgradeSchema.php

```diff
@@ -59,8 +59,8 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
             $this->updateSchemaVersion204($setup);
         }
 
-        if (version_compare($context->getVersion(), '2.0.7', '<')) {
-            $this->updateSchemaVersion207($setup);
+        if (version_compare($context->getVersion(), '2.1.0', '<')) {
+            $this->updateSchemaVersion210($setup);
```

Comment:
Why do you change the version you can keep this on 2.0.7. Changing it could lead to problems for developers using the develop branch.